aries-site/src/examples/components/namevaluelist/Examples/NameValueListDefaultExample.js

```diff
@@ -16,7 +16,7 @@ export const NameValueListDefaultExample = () => (
         }
         return (
           <NameValuePair key={name} name={name}>
-            {name === 'Created on' ? <> {date}</> : <>{value}</>}
+            {name === 'Created on' ? <> {date}</> : value}
```

Comment:
confirming that this is a "no useless fragments" rule?

Reply:
but in this case I don't think the fragment is even doing anything

Reply:
I'm in favor of this rule